src/SmartComponents/modules/reducers.js

```diff
@@ -88,6 +89,70 @@ function selectedSystems(selectedIds, selectedSystem) {
     return selectedIds;
 }
 
+function convertFactsToCSV(data, systems) {
+    let result;
+    let keys;
+    let columnDelimiter;
+    let lineDelimiter;
+
+    if (data === null || !data.length) {
+        return null;
+    }
+
+    columnDelimiter = data.columnDelimiter || ',';
+    lineDelimiter = data.lineDelimiter || '\n';
+
+    result = 'Fact,State,';
+    result += systems.join(columnDelimiter);
+    result += lineDelimiter;
+
+    keys = Object.keys(data[0]);
+
+    data.forEach(function(fact) {
+        keys.forEach(function(key, index) {
+            if (index > 0) {
+                result += columnDelimiter;
+            }
+
+            if (key === 'systems') {
+                fact[key].forEach(function(system) {
+                    result += system.value;
+                    result += columnDelimiter;
+                });
+                result += lineDelimiter;
+            } else {
+                result += fact[key];
+            }
+        });
+    });
+
+    return result;
+}
+
+function downloadCSV(driftData, systems) {
+    let data;
+    let filename;
+    let link;
+    let csv = convertFactsToCSV(driftData, systems);
+
+    if (csv === null) {
+        return;
+    }
+
+    filename = 'export.csv';
```

Comment:
Maybe name it `system-comparison-export-%date.csv` so they can export them easily without clashes and know what the export is for?